language/tools/move-build/tests/test_sources/parsing/full_manifest_with_extra_field.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "name"
+version = "0.1.2"
+license = "license"
+authors = ["some author"]
+
+[addresses]
+Std = "_"
+CoreAddr = "0x1"
+C = "0x1"
+OtherD = "_"
+
+[dev-addresses]
+DevStd = "0x2"
+DevCoreAddr = "0x1"
+
+[build]
+source_version = "0.0.0"
+extra_field = true
+
+[dependencies]
+A = { local = "../a" }
+B = { local = "../b", addr_subst = { "a" = "0xDEADBEEF" }, version = "0.0.1" }
+C = { local = "../some_path", addr_subst = { "a" = "0xCAFE", "d" = "0x4" }, version = "0.0.2" }
```

Comment:
unrelated to this test in particular, but any final thoughts on banning assignment here? Since the addresses are opaque, might make sense to move this up into the addresses section? or do we only want that for exported addresses?

Reply:
I think keeping assignments here makes the most sense to me after working on the resolution stuff. The main thing that I encountered there was that keeping local renames local may not actually be what we want.

In particular I think what we really want is "this package P imports "A" from C and "A" from D, and exports addresses "CA" and "DA" from it (and no "A"'s are -- unless P declares a named address "A" itself) so that you can refer to those different A's in other packages that import P".

Reply:
Works for me, but I think a lot of these small details will be easy to sort out in examples later. Should be easy enough to change